Index: chromium-139.0.7258.5/components/country_codes/country_codes.cc
===================================================================
--- chromium-139.0.7258.5.orig/components/country_codes/country_codes.cc
+++ chromium-139.0.7258.5/components/country_codes/country_codes.cc
@@ -135,7 +135,7 @@ CountryId GetCurrentCountryID() {
 #elif BUILDFLAG(IS_POSIX) || BUILDFLAG(IS_FUCHSIA)
 
 CountryId GetCurrentCountryID() {
-  const char* locale = setlocale(LC_MESSAGES, nullptr);
+  const char* locale = getenv("LANG");
   if (!locale) {
     return CountryId();
   }
