--- a/components/viz/host/gpu_host_impl.cc
+++ b/components/viz/host/gpu_host_impl.cc
@@ -20,9 +20,7 @@
 #include "gpu/ipc/common/gpu_client_ids.h"
 #include "gpu/ipc/host/gpu_disk_cache.h"
-#include "gpu/webgpu/dawn_commit_hash.h"
 #include "mojo/public/cpp/bindings/sync_call_restrictions.h"
 #include "services/webnn/host/weights_file_provider.h"
 #include "skia/buildflags.h"
-#include "skia/ext/skia_commit_hash.h"
 #include "ui/gfx/font_render_params.h"

@@ -108,5 +108,5 @@  // - Skia's git hash is because some cached shaders might not be used in a
   // newer version of Skia.
-  return SKIA_COMMIT_HASH "_" DAWN_COMMIT_HASH;
+  return "what_ever";
 }
 #endif
